store/src/styles/components/kiosk.scss

```
@@ -128,7 +128,7 @@
    top: 0;
    left: 0;
    width: 100%;
-   height: 100%;
+   height: fit-content;
```

```suggestion
   height: 100%;
```
Making it fit-content it will add a vertical scroll

---

@FAsami Thanks for that. The task was to make the text on the image bigger, thought making the height of the image fit-content would make the "Only digital text bigger..", with the height 100%, the image on fulfilment screen becomes vertically compressed and causes the text to be smaller. How would you suggest we make the text bigger on the image without making veritical scroll?

---

OK what you were saying has been noted. So I was getting the resolution wrong. Thanks, am committing your suggestion. @FAsami